src/session_impl.cpp

```diff
@@ -6865,6 +6950,7 @@ namespace aux {
 			, struct tracker_response const& resp)
 		{
 			TORRENT_UNUSED(tracker_ips);
+			if (!should_log()) return;
```

Comment:
It looks like the construction of the tracker_logger is already protected by ``should_post<>()``, I suppose you could use ``should_log()`` instead, for symmetry. and those checks inside ``tracker_logger`` itself can be removed.